cosmotheka/mappers/mapper_ACT_compsept.py

```diff
@@ -26,7 +26,6 @@ def _get_signal_map(self):
             signal_map,
             nside=self.nside,
             lmax=self.lmax,
-            niter=3
```

Comment:
Not sure what this exactly does and what is the best option but maybe we can use here the same as for the `NmtField`? If so, I would read it from the config file. I think `self.config['sphere']['n_iter_sht']`

Reply:
sorry, i got confused. the `master.yml` has the `sphere` section, but for the mapper i think you should pass `n_iter_sht` directly, like for the nside and coords